source/js/buyers-guide/related-articles.js

```diff
@@ -0,0 +1,21 @@
+const RelatedArticles = {
+  init: () => {
+    const firstParagraph = document.querySelector(
+      ".paragraph-block .rich-text"
+    );
+    const relatedContainer = document.querySelector("#pni-related-content");
+    if (!firstParagraph || !relatedContainer) return;
+    const relatedContent = relatedContainer.querySelector("div");
+    const relatedPlace = firstParagraph.querySelector("*:nth-child(3)");
+    if (!relatedPlace) return;
+    firstParagraph.insertBefore(relatedContent, relatedPlace);
+    relatedContent.classList.add(
+      "large:tw-float-right",
+      "tw-p-4",
+      "tw-pr-0",
+      "large:tw-mr-[-20%]"
+    );
```

Comment:
What do you think about adding a method name that describes what it does? `RelatedArticles.init()` does tell me that it's about the related articles but not what to expect from happening. What do you think about something like `floatRelatedArticlesNextToFirstParagraph`?

Reply:
addressed

Reply:
Nice. Thanks. That makes it a lot easier to understand what's happening.